qadence/operations.py

```diff
@@ -878,9 +899,9 @@ def __init__(self, control: int | tuple[int, ...], target1: int, target2: int) -
         if isinstance(control, tuple):
             control = control[0]
 
-        a00m = 0.5 * (Z(control) - I(control))
+        a00m = -N(control)
```

Comment:
@jpmoutinho Can we do better here by introducing more number operators ? It feels so but I haven't found a godd way so far.

Reply:
You mean to simplify all the lines here? The `Z+I` is the `|0><0|` projector, but not sure it's worth adding more operators just to simplify that if they won't be used in other places. I like keeping the Pauli's explicit.

Reply:
What I meant was that since I integrated `N` in places, it feels here that it is halfway backed. I would replace with `N` wherever possible (say `a00m` and `a22` here) for consistency and also because it makes it cleaner I think ?